### PR TITLE
Subscription table implemented. 

### PR DESCRIPTION
**Pull request by someshfengde**
## Description

Please include a summary of the change and the issue it solves. 

Fixes #issue_number

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] 📄 This change requires a documentation update

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

https://github.com/mindsdb/mindsdb/assets/42097653/8fc19f65-5d75-439d-8a99-7257994fb910

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by someshfengde** [comment]
#7765

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by martyna-mindsdb** [comment]
@someshfengde 
Please sign our CLA so we can merge your PR after reviewing.

File: mindsdb/integrations/handlers/paypal_handler/README.md [hunk 1]
```diff
@@ -85,6 +91,11 @@ Query Invoices_table:
 SELECT * FROM paypal_datasource.invoices
 ~~~~
 
+Query Subscriptions_table:
+~~~~sql
+SELECT * FROM paypal_datasource.subscription
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/paypal_handler/README.md:96]: For the sake of consistency, let's update the table name to **subscriptions** (instead of subscription), throughout the code.

File: mindsdb/integrations/handlers/paypal_handler/paypal_handler.py [hunk 2]
```diff
@@ -38,6 +38,9 @@ def __init__(self, name: str, **kwargs):
 
         invoices_data = InvoicesTable(self)
         self._register_table("invoices", invoices_data);
+    
+        subscripotion_data = SubscriptionTable(self)
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/paypal_handler/paypal_handler.py:42]: There is a typo in this variable name - it should be **subscriptions_data** throughout the code.

**Comment by someshfengde** [comment]
@martyna-mindsdb did the changes

**Comment by martyna-mindsdb** [comment]
@someshfengde 
In the README file, please update the parameters as below:
```
...
  "client_id": "your-client-id",
  "client_secret": "your-client-secret"
```

**Comment by someshfengde** [comment]
Sorry totally forgot about that one

**Review by martyna-mindsdb — APPROVED** [review]

**Comment by martyna-mindsdb** [comment]
@someshfengde 
Before we can merge your PR, please sign our CLA (as mentioned above) by leaving a comment here: I have read the CLA Document and I hereby sign the CLA

**Comment by someshfengde** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by someshfengde** [comment]
recheck

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by tomhuds** [comment]
@martyna-mindsdb @someshfengde @ZoranPandovski before merging please ensure tests have been added and pass

**Comment by martyna-mindsdb** [comment]
@tomhuds 
Please specify what tests you mean.

This kind of issues (for extending integrations with new commands or tables) do not usually include written tests, like [this one](https://github.com/mindsdb/mindsdb/pull/7821) too.

**Comment by tomhuds** [comment]
They should include unit tests (none currently for [shopify] (https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler) or [paypal](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/paypal_handler))

Also integration tests (none currently for [shopify/paypal](https://github.com/mindsdb/mindsdb/tree/staging/tests/handler_tests)
